src/packages/quote.py

```diff
@@ -1,110 +1,125 @@
+		quotes, msg = sorted(quotes, key=lambda i: i['id']), ''
 		if len(quotes) == 0:
 			return await ctx.send(embed = discord.Embed(title="No quotes could be found", description="Try a different search term or submit your own using !q add", color=Colour.from_rgb(255,0,0)))
 
-		startingid = quotes[0]['id']
-		quoteAbles = []
-		for quote in quotes:  # split quotes up into fields for the embed
-			text_quote = f"{quote['id']}: \"{quote['quote']}\" - {quote['author']}\n"
-			if len(qmsg) + len(text_quote) >= 975:
-				quoteAbles.append({"msg": qmsg, "start": startingid, "prev": previousid})
-				qmsg, startingid = '', quote['id']
-			qmsg, previousid = qmsg + text_quote, quote['id']
+		starting_id = quotes[0]['id']
+		quoteables = []
 
-		quoteAbles.append({"msg": qmsg, "start": startingid, "prev": previousid})
-		await self.mass_quote_embed(ctx, quoteAbles)
+		for quote in quotes:
+			sub_msg = f"{quote['id']}: \"{quote['quote']}\" - {quote['author']}\n"
+			if len(msg) + len(sub_msg) >= 975:
+				quoteables.append({"msg": msg, "start": starting_id, "prev": previous_id})
+				msg, starting_id = '', quote['id']
+			msg, previous_id = msg + sub_msg, quote['id']
 
-	async def mass_quote_embed(self, ctx: commands.Context, quoteFields: List[Dict[str, Union[int, str]]]) -> None:
+		quoteables.append({"msg": msg, "start": starting_id, "prev": previous_id})
+		await self.mass_quote_embed(ctx, quoteables)
+
+	# Displays quotes en-mass
+	async def mass_quote_embed(self, ctx: commands.Context, quote_blocks: List[Dict[str, Union[int, str]]]) -> None:
 		colours = [Colour.from_rgb(255,0,0), Colour.orange(), Colour.gold(), Colour.green(), Colour.blue(), Colour.dark_blue(), Colour.purple()]
 		embed = discord.Embed(title="Quotes", colour = colours[0])
-		for index, value in enumerate(quoteFields):
+
+		for index, value in enumerate(quote_blocks):
 			if index % 6 == 0 and index != 0:
 				embed.set_footer(text=f"Powered by {self.bot.user.name}")
 				await ctx.send(embed=embed)
 				embed = discord.Embed(title="Quotes", colour = colours[(index // 6) % len(colours)])
 			embed.add_field(name=f"Quotes {value['start']} : {value['prev']}", value=value['msg'], inline=False)
 		embed.set_footer(text=f"Powered by {self.bot.user.name}")
-		await ctx.send(embed=embed)
-
 
-	async def update(self) -> None:
-		self.config = self.load_config()
-		log.info(f'Quotes ran an update')
+		await ctx.send(embed=embed)
 
+	# Command group !quote
 	@commands.group(aliases=['quote'], brief='Subgroup for quote functionality', description='Subgroup for quote functionality. Use !help q')
 	async def q(self, ctx: commands.Context) -> None:
 		if not path.isfile('storage/db/quotes/' + str(ctx.guild.id) + '.json'):
 			with open('storage/db/quotes/' + str(ctx.guild.id) + '.json', 'w+', encoding='utf-8') as file:
 				json.dump({}, file, indent=4)
 		if ctx.invoked_subcommand is None:
-			log.debug(f'QUOTE User {ctx.author.name} has passed an invalid quote subcommand: {ctx.message.content}')
```

Comment:
There's a couple of these around where I set them to debug mode and you changed the message. Would like them to be debug mode again, currently using info to just display what command was passed. Whats your opinion?

Reply:
I'd argue showing every command is more verbose than just the ones that result in an error, so it should be the other way around?

Reply:
wdym julia? I get what maurits said, And I changed the "QUOTE" msg, simply because it was inconsistent with the rest of the logging. Id say that info should only display the command getting called and any errors/warnings. Things like what kind of arguments the user passed is something youd only use while debugging imo.

Also about the imports, Im ok with moving .env loading / logging setup outside of imports, but I sorted the imports on alphabet. Is that not how vscode does it?

Reply:
If I understand what Maurits said correctly, we're printing every command as info but when a command fails it's a debug log. Feels a bit counterintuitive to me, as printing *every* command is more verbose and something we don't need most of the time.

I'd say maybe use warning for invalid commands, error for failed ones (when it's our fault), and info for a full log of every command that has been executed (and then optionally debug for extra context etc)

Reply:
@Bluerberry vscode does something weird, I'm chill with alphabet sorting maybe split it into just `import x` and `import y from x` to keep it more cleaner?

Reply:
Could always use an autoformatter? 👀

Kinda wanna set up a github action that enforces code styling and type checking now

Reply:
or or or, the best way to sort imports: on line-length. then it looks organized

Reply:
@juliavdkris no worries, if it breaks its still logged via main.py on_command_error (yes, it now just doesn't `pass` and return. tho the logger is kinda shite at logging a stacktrace) most of the logs in quote can be debug yet are info now. This was the debug command to check that we are sending a random quote when there is no subcommand, it can be debug now safely.

Reply:
yes but that is pain to search through lmao, I won't stop you tho, aesthetics

Reply:
Ahhh alright cool, then I'm with you

Reply:
I do see some inconsistencies especially with the "user executed this command" logging, but I'll open a separate issue for that